tests/brun/path-5.txt

```diff
@@ -0,0 +1,3 @@
+brun -n -c '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001' '(((0x1337 . (0x1337 . (42 . 0x1337))) . 0x1337) . 0x1337)'
+cost = 53
```

Comment:
@richardkiss You may notice that leading zeros don't seem to cost anything here.

Reply:
Hmm, they ought to cost a minimal amount, like 1. I thought I'd implemented that.

Reply:
I see, I think I implemented an additional cost of leading zeros for rust, but not for python. I prefer how rust does it. We need to unify this algorithm. This test will help catch the discrepancy.